pkg/storage/bucket/object_client_adapter.go

```diff
@@ -0,0 +1,143 @@
+package bucket
+
+import (
+	"context"
+	"io"
+	"strings"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/pkg/errors"
+
```

Comment:
Wonder why the linter did not catch this empty newline. Please run `make format`